src/test/kotlin/pt/up/fe/ni/website/backend/controller/AccountControllerTest.kt

```diff
@@ -653,60 +692,127 @@ class AccountControllerTest @Autowired constructor(
-                                "linkedin" to newLinkedin,
-                                "github" to newGithub,
-                                "websites" to newWebsites
-                            )
-                        )
-                    )
-            ).andExpectAll(
-                status().isOk,
-                content().contentType(MediaType.APPLICATION_JSON),
-                jsonPath("$.name").value(newName),
-                jsonPath("$.email").value(newEmail),
-                jsonPath("$.bio").value(newBio),
-                jsonPath("$.birthDate").value(newBirthDate.toJson()),
-                jsonPath("$.photoPath").value(newPhotoPath),
-                jsonPath("$.linkedin").value(newLinkedin),
-                jsonPath("$.github").value(newGithub),
-                jsonPath("$.websites.length()").value(1),
-                jsonPath("$.websites[0].url").value(newWebsites[0].url),
-                jsonPath("$.websites[0].iconPath").value(newWebsites[0].iconPath)
-            ).andDocument(
-                documentation,
-                "Update accounts",
-                "Update a previously created account, with the exception of its password, using its ID.",
-                urlParameters = parameters,
-                documentRequestPayload = true
+            val data = objectMapper.writeValueAsString(
+                mapOf(
+                    "name" to newName,
+                    "email" to newEmail,
+                    "bio" to newBio,
+                    "birthDate" to newBirthDate,
+                    "linkedin" to newLinkedin,
+                    "github" to newGithub,
+                    "websites" to newWebsites
+                )
+            )
+
+            mockMvc.multipartBuilder("/accounts/${testAccount.id}")
+                .addPart("dto", data)
+                .asPutMethod()
+                .perform()
+                .andExpectAll(
+                    status().isOk,
+                    content().contentType(MediaType.APPLICATION_JSON),
+                    jsonPath("$.name").value(newName),
+                    jsonPath("$.email").value(newEmail),
+                    jsonPath("$.bio").value(newBio),
+                    jsonPath("$.birthDate").value(newBirthDate.toJson()),
+                    jsonPath("$.linkedin").value(newLinkedin),
+                    jsonPath("$.github").value(newGithub),
+                    jsonPath("$.websites.length()").value(1),
+                    jsonPath("$.websites[0].url").value(newWebsites[0].url),
+                    jsonPath("$.websites[0].iconPath").value(newWebsites[0].iconPath)
+                )
+//                .andDocument(
+//                    documentation,
+//                    "Update accounts",
+//                    "Update a previously created account, with the exception of its password, using its ID.",
+//                    urlParameters = parameters,
+//                    documentRequestPayload = true
+//            )
```

Comment:
```suggestion
```

Reply:
There are multiple lines commented, is that intentional?

Reply:
Yes, because multipart documentation is not implementaed, but I didn't want to delete the documentation written by other people